build.gradle.kts

```diff
@@ -33,7 +35,7 @@ dependencies {
         "api",
         "core",
         "slf4j-impl"
-    ).forEach { compile("org.apache.logging.log4j:log4j-$it:2.10.0") }
+    ).forEach { compile("org.apache.logging.log4j:log4j-$it:$log4jVersion") }
```

Comment:
The `compile` dependency becomes part of the POM, which is part of the API, so we should log this change.

Reply:
Done, thanks.